### PR TITLE
Correct order of magnitude of packages on opam

### PR DESCRIPTION
**Pull request by shonfeder**
## Changes Made

Opam currently has around 3.5k packages! 🎉  So this tiny wording change will give prospective users a more accurate sense of the current amount of libraries available.

- [x] PR is descriptively titled and links the original issue above
- [x] Context for what motivated the change (if this is a change to some content)

### CODE REVIEW

**Comment by patricoferris** [comment]
Thanks @shonfeder, sounds good to me. Whilst we're at it, probably would make sense to update the [French as well](https://github.com/ocaml/ocaml.org/blob/master/site/index.fr.md). From "de centaines de paquets" to "de milliers de paquets"... I think?

**Comment by Octachron** [comment]
Yes, "de milliers de paquets" is the correct translation.

**Comment by shonfeder** [comment]
Changed in the french. Hopefully correctly! Thanks for the language and pointer.

File: site/index.fr.md [hunk 1]
```diff
@@ -55,7 +55,7 @@
 					</a>
                     <h1><a href="https://opam.ocaml.org">Contributions</a></h1>
                     <p>Le gestionnaire de paquets <a href="https://opam.ocaml.org">OPAM</a> vous donne accès aux multiples versions de
-					<a href="https://opam.ocaml.org/packages/">centaines de paquets</a>.</p>
+					<a href="https://opam.ocaml.org/packages/">de milliers de paquets</a>.</p>
```
Comment (Octachron) [inline site/index.fr.md]: Sorry, I included the article "de" ("of") in my quoted text, but the article is already present before in  `aux multiples versions de <a href="...">centaines..."`. The correct sentence would be:
>```
> vous donne accès aux multiples versions de
> <a href="https://opam.ocaml.org/packages/">milliers de paquets</a>.</p>
>```

Comment (shonfeder) [reply site/index.fr.md]: Ah my bad for not looking carefully. Thanks for the correction.

Comment (shonfeder) [inline site/index.fr.md]:
```suggestion
					<a href="https://opam.ocaml.org/packages/">milliers de paquets</a>.</p>
```

**Comment by fccm** [comment]
You changed 'centaines' by 'de milliers', there is another 'de' before the link, so it's now duplicated.

**Comment by shonfeder** [comment]
I don’t see duplicated ‘de’ in the diff. But I am also not a French speaker. I suggest a French speaker make the trivial change in whatever way is needed to make this trivial update acceptable.

**Comment by redianthus** [comment]
Currently it's: `Le gestionnaire de paquets OPAM vous donne accès aux multiples versions de milliers de paquets.` which I can confirm to be correct.

**Comment by fccm** [comment]
Indeed it has been fixed by the second commit `Update site/index.fr.md` dbbc7d0
Everything's fine now.

**Comment by patricoferris** [comment]
Thanks @shonfeder and everyone else :))

**Pull request merged**